tests/test_validators.py

```diff
@@ -51,3 +62,56 @@ def test_valid(self):
     def test_invalid(self):
         with self.assertRaisesMessage(ValidationError, "Item b's value None is not of type int."):
             DictValuesTypeValidator(int)({"a": 42, "b": None})
+
+
+class FileValidatorTestCase(SimpleTestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        fd, cls.temporary_file_name = mkstemp()
+
+    @classmethod
+    def tearDownClass(cls):
+        os.remove(cls.temporary_file_name)
+
+    def test_valid_read(self):
+        FileValidator()(self.temporary_file_name)
+
+    def test_valid_write(self):
+        FileValidator(mode="w")(self.temporary_file_name)
+
+    @patch("appsettings.validators.open", side_effect=IOError(errno.ENOENT, "Not a file"))
```

Comment:
This could be easily mocked by creating an empty temporary directory.

Reply:
Done.